### PR TITLE
proper debug logs for aiohttp's StreamResponse

### LINKED ISSUE #786: Aiohttp StreamResponse responses don't work
### Description
Aiohttp StreamResponse responses don't work with Connexion.
`AttributeError: 'StreamResponse' object has no attribute 'body'`

### Expected behaviour
You can use streaming responses.

### Actual behaviour
It crashes with an exception.
```
ERROR:aiohttp.server:Error handling request
Traceback (most recent call last):
  File "/usr/local/lib/python3.7/site-packages/aiohttp/web_protocol.py", line 390, in start
    resp = await self._request_handler(request)
  File "/usr/local/lib/python3.7/site-packages/aiohttp/web_app.py", line 366, in _handle
    resp = await handler(request)
  File "/usr/local/lib/python3.7/site-packages/aiohttp/web_middlewares.py", line 106, in impl
    return await handler(request)
  File "/usr/local/lib/python3.7/site-packages/aiohttp/web_middlewares.py", line 106, in impl
    return await handler(request)
  File "/usr/local/lib/python3.7/site-packages/connexion/apis/aiohttp_api.py", line 32, in oauth_problem_middleware
    response = yield from handler(request)
  File "/usr/local/lib/python3.7/site-packages/connexion/decorators/coroutine_wrappers.py", line 23, in wrapper
    connexion_response = yield from connexion_response
  File "/usr/local/lib/python3.7/site-packages/connexion/apis/aiohttp_api.py", line 212, in get_response
    response.status, extra={'data': response.body, 'url': url})
AttributeError: 'StreamResponse' object has no attribute 'body'
ERROR:aiohttp.server:Unhandled exception
Traceback (most recent call last):
  File "/usr/local/lib/python3.7/site-packages/aiohttp/web_protocol.py", line 410, in start
    await resp.prepare(request)
  File "/usr/local/lib/python3.7/site-packages/aiohttp/web_response.py", line 300, in prepare
    return await self._start(request)
  File "/usr/local/lib/python3.7/site-packages/aiohttp/web_response.py", line 608, in _start
    return await super()._start(request)
  File "/usr/local/lib/python3.7/site-packages/aiohttp/web_response.py", line 367, in _start
    await writer.write_headers(status_line, headers)
  File "/usr/local/lib/python3.7/site-packages/aiohttp/http_writer.py", line 110, in write_headers
    self._write(buf)
  File "/usr/local/lib/python3.7/site-packages/aiohttp/http_writer.py", line 67, in _write
    raise ConnectionResetError('Cannot write to closing transport')
ConnectionResetError: Cannot write to closing transport
```

### Steps to reproduce
```python3
response = web.StreamResponse(status=200)
await response.prepare(request)
await response.drain()
await response.write(b'lubie placki')
await response.write_eof()
return response
```

### Additional info:

Output of the commands:

- `python --version`
Python 3.7.1

- `pip show connexion | grep "^Version\:"`
Version: 2018.0.dev1

### PR DESCRIPTION
**Pull request by szborows**
Fixes #786 .

Changes proposed in this pull request:

 - fixed an issue
 - the code could be refactored a bit too, but leaved it to be reviewed and commented by connexion contributor

### CODE REVIEW

File: connexion/apis/aiohttp_api.py [hunk 1]
```diff
@@ -208,8 +208,12 @@ def get_response(cls, response, mimetype=None, request=None):
         if isinstance(response, ConnexionResponse):
             response = cls._get_aiohttp_response_from_connexion(response, mimetype)
 
-        logger.debug('Got data and status code (%d)',
-                     response.status, extra={'data': response.body, 'url': url})
+        if isinstance(response, web.StreamResponse):
```
Comment (jmcs) [inline connexion/apis/aiohttp_api.py]: Can you make this an `elif`?

Comment (szborows) [reply connexion/apis/aiohttp_api.py]: I'm assuming that `ConnexionResponse` never wraps `web.StreamResponse`. If I change this to elif there's additional debug print that was not there before.

Is this intentional?

What are possibilities here by the way?

- `web.Response`
- `web.StreamResponse`
- `ConnexionResponse`

anything else?

Comment (szborows) [reply connexion/apis/aiohttp_api.py]: @jmcs can you please look into this? I want to know how to proceed.

Comment (jmcs) [reply connexion/apis/aiohttp_api.py]: NVM. You're right.

**Comment by jmcs** [comment]
:+1:

**Pull request merged**